### DISCUSSION TITLE
Layers in overlay

### DISCUSSION THREAD

**Discussion by stofmania** [bozdoz/wp-plugin-leaflet-map#238] [General]
Hi,

I would like to have this in base layer : https://tile.gbif.org/3857/omt/{z}/{x}/{y}@1x.png?style=gbif-natural-fr
It's not a problem with "tileurl"

But, I would like to add another layer in overlay : https://api.gbif.org/v2/map/occurrence/density/{z}/{x}/{y}@1x.png?srs=EPSG:3857&bin=hex&hexPerTile=41&taxonKey=7719048&style=classic.poly

This feature seems to not be implemented in this plugin (I haven't found the solution).

Do you think it's possible to add this feature ?
Thanks in advance :)

An example of the 2 layers :
![image](https://github.com/bozdoz/wp-plugin-leaflet-map/assets/146885157/edd95650-6067-4a8f-9ee3-b152291c96a7)

**Comment by bozdoz** [comment]
Probably requires either of these open issues to be resolved:

https://github.com/bozdoz/wp-plugin-leaflet-map/issues?q=is%3Aissue+is%3Aopen+overlay

**Reply by stofmania** [reply]
Thanks for your answer.
Do you think itt's possible to solve one of this issue soon ?
Thanks in advance ?

**Reply by bozdoz** [reply]
Very unlikely.  They've been open since 2018 without a clear need or resolution.  Also, as time goes on it's been much harder to maintain this plugin.  Contributions welcome, and if you wanted to post your use case to one of the issues, it may help define the "need" for them.

**Reply by bozdoz** [reply]
This one in particular saw a lot of discussion, but no clear answers: https://github.com/bozdoz/wp-plugin-leaflet-map/issues/54
